e2e/colorpicker.e2e-spec.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  element,
+  by
+} from 'protractor';
+
+import {
+  expect,
+  SkyHostBrowser
+} from '@skyux-sdk/e2e';
+
+describe('Colorpicker', () => {
+  beforeEach(() => {
+    SkyHostBrowser.get('visual/colorpicker');
+    SkyHostBrowser.setWindowBreakpoint('lg');
+  });
+
+  it('should match previous colorpicker screenshot', (done) => {
+    expect('#screenshot-colorpicker').toMatchBaselineScreenshot(done);
+  });
```

Comment:
Mind throwing some `xs` screenshots in here?

Reply:
done